scripts/check_driver.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# https://github.com/aquadzn
+# ```bash check_driver.sh```
+
+
+check_codec() {
+    if $(vainfo |& grep -q -e "H265" -e "HEVC")
+    then
+        echo -e "\n- H265 / HEVC found."
+    else
+        echo -e "\nH265 or HEVC not found."
+    fi
+
+    if $(vainfo |& grep -q "H264")
+    then
+        echo -e "- H264 found."
+    else
+        echo -e "\nH264 not found.\nPlease generate a report with the following command:\n\t- curl https://raw.githubusercontent.com/NicolasGuilloux/blade-shadow-beta/master/scripts/report.pl | perl"
```

Comment:
`Sadly, not supporting at least H264 means not supporting Shadow at all. To get some help, please generate a full report with the following command, and share it with the Discord community`